www/js/app.js

```diff
@@ -55,10 +52,44 @@ var app = angular.module('starter',
                 url:'/settings',
                 templateUrl: 'templates/settings.html',
                 controller: 'Settings'
+            })
+            .state('profile', {
+                url:'/profile',
+                templateUrl: 'templates/profile.html',
+                controller: 'Profile'
             });
         $urlRouterProvider.otherwise('/');
     }) 
     
+    .directive('profileBarWidget', function($ionicGesture) {
+        return {
+            restrict: 'AE',
+            // declare the directive scope as private (and empty)
+            scope: {},
+            // add behaviour to our buttons and use a variable value
+            template:   '<div class="thumbnail">'+
```

Comment:
Please move the template out to an HTML partial (check the `templates` folder, or consider making a new `directives` folder) and then use a `templateUrl` to reference it